Dockerfile

```diff
@@ -1,8 +1,19 @@
 # Start runtime image,
-FROM amsterdam/python:3.9-buster
+FROM amsterdam/python:3.9-slim-buster
+
+RUN apt update
+RUN apt-get update \
+ && apt-get autoremove -y \
+ && apt-get install --no-install-recommends -y \
+        libpq-dev \
+        python-dev \
+        gcc \
+        git
 
 WORKDIR /app
 COPY . ./
 RUN pip install -e ".[django,tests]"
+# So we can use local schemas
+RUN git clone https://github.com/Amsterdam/amsterdam-schema.git /tmp/ams-schema
```

Comment:
wordt dit bij iedere build uitgevoerd of blijft dat gecached als je een rebuild doet?

Reply:
Dit wordt zo te zien gecachet tot er een wijziging in de code is (`COPY . ./` hierboven).